jupyter-extension/jupyterlab_pachyderm/pfs_manager.py

```diff
@@ -270,59 +283,19 @@ def dir_exists(self, path):
         return response.file_type == pfs.FileType.DIR
 
     def exists(self, path) -> bool:
-        file = self._get_file_from_path(path)
+        file = self._get_file_from_path(path, self.mounted_branch)
```

Comment:
self._get_file_from_path only had one argument. Either you meant to add a second argument for the branch or this need to be removed.

Reply:
I think the issue here is `DatumManager._get_file_from_path` has no branch argument, but `PFSManager._get_file_from_path` does have a branch argument (which makes sense the datum view is not considering the selected branch from the explore view).

Reply:
Ignore my last comment here, just need to remove this! I was looking at master and got confused.